<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>addfood</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>images/m.jpg</normaloff>images/m.jpg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>images/图标4.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>40</y>
      <width>381</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线 Light</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>猫猫投喂登记——新增食物</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线 Light</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>食物名称</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_name">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>150</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线 Light</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_type">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>250</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线 Light</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_type">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>250</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线 Light</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>新增食物种类</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_manufacturer">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>350</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线 Light</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>新增生产厂家</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_manufacturer">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>350</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线 Light</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>350</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线 Light</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>生产厂家</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbox_manufacturer">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>350</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线 Light</family>
      <pointsize>11</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线 Light</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>食物种类</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbox_type">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>250</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线 Light</family>
      <pointsize>11</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_add">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>450</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线 Light</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>新增食物</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>70</y>
      <width>191</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>images/罐头1.png</pixmap>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
